apps/web/src/app/app/(auth)/login/layout.tsx

```diff
@@ -24,7 +24,7 @@ export default async function AuthLayout({
             className="rounded-full border border-border"
           />
         </Link>
-        <div className="mx-auto flex w-full max-w-lg flex-1 flex-col justify-center gap-8 text-center md:mx-0 md:text-left">
+        <div className="mx-auto flex w-full max-w-lg flex-1 flex-col justify-center gap-8 text-center md:mx-auto md:text-left">
```

Comment:
```suggestion
        <div className="mx-auto flex w-full max-w-lg flex-1 flex-col justify-center gap-8 text-center md:text-left">
```

I think we can even delete it entirely as the first utility class already is `mx-auto`.

Reply:
Thanks for pointing that out.
I have removed extra mx:auto